russstation/code/modules/uplink/uplink_items.dm

```diff
@@ -13,3 +13,13 @@
 	cost = 7
 	limited_stock = 2
 	restricted_roles = list("Janitor")
+
+/datum/uplink_item/role_restricted/laughter_pen
+	name = "Deadly Laughter Pen"
+	desc = "A syringe disguised as a functional pen, filled with a potent mix of drugs, including a \
+			chemical that causes uncontrollable laughter and a toxin that deprives the body of oxygen. \
+			The pen holds one dose of the mixture, and can be refilled with any chemicals. Note that before the target \
+			falls asleep, they will be able to move and act."
+	item =  /obj/item/pen/laughter
+	cost = 4
+	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
```

Comment:
This assignment to `exclude_modes` is redundant as its superclass `/datum/uplink_item/role_restricted` already defines it with the same value. Also, would it make sense to make this a clown-specific uplink item?

Reply:
Yea, I forgot to do that. It's clown specific now.